playbooks/ufw-playbook.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+- hosts: frontend_servers
+  roles:
+    - franklinkim.ufw
+  become: yes
+  vars:
+    ufw_rules:
+      - { port: 80, rule: allow }
+      - { port: 443, rule: allow }
+      - { from_ip: '127.0.0.1/8' }
```

Comment:
Just for my own understanding, does this line mean "allow everything from localhost"?  I'm guessing "rule: allow" is the default so it can simply be omitted, but I noticed you had it for the two rules above so I was curious if this one was different...